### PR TITLE
[RFC][NI]New contexts colors to address new partials 

### PR DESCRIPTION
**Pull request by fredcerdeira**

### CODE REVIEW

File: src/styles/_common/_colors.scss [hunk 1]
```diff
@@ -35,7 +35,9 @@ $palette: (
     'modal': #6b6b6b,
     'header': #545d69,
     'close': #b2b9c2,
-    'secondary': rgba(0, 60, 88, 0.7)
+    'secondary': rgba(0, 60, 88, 0.7),
+    'sorting-items': #788392,
+    'helper-copy': 788392
```
Comment (telmaantunes) [inline src/styles/_common/_colors.scss]: no hash needed? 'helper-copy': #788392

Comment (telmaantunes) [inline src/styles/_common/_colors.scss]: and in the same group, "text", 2 colors equal but with different names, is also good?

File: src/styles/_common/_colors.scss [hunk 2]
```diff
@@ -128,6 +130,15 @@ $palette: (
     'active': #66d5ff,
     'item-hover': #dbf5ff
   ),
+  'arrows':(
+    'item-panel': #949da9,
+    'item-panel-hover': #00222f,
+    'toggle-accordion': #788392,
+    'sort-item': #788392
```
Comment (telmaantunes) [inline src/styles/_common/_colors.scss]: same as above

File: src/styles/_common/_colors.scss (hunk 1 above)
Comment (fredcerdeira) [inline src/styles/_common/_colors.scss]: @telmaantunes because we are using a context logic, some colors will be repeated.

**Pull request merged**